modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -42,6 +42,8 @@ GrainTracker::GrainTracker(const InputParameters & parameters) :
 {
   if (!_is_elemental && _compute_op_maps)
     mooseError("\"compute_op_maps\" is only supported with \"flood_entity_type = ELEMENTAL\"");
+
+  _empty_2.resize(_n_vars, 0);
```

Comment:
You tell me, I think zero is fine here as you really shouldn't hit this case unless you forget to enable computation of the op maps (minor error) or there is some serious logic bug. In either case if you get `libMesh::invalid_uint` and you end up indexing into a vector with that, you'll get a seg fault versus silent failure by indexing with a zero.